css/creative.css

```diff
@@ -4505,3 +4505,44 @@ align-items: center;
   width: 50%;
   margin: 0 20px
 }
+
+.footer-logo-container {
+  padding: 20px;
+  border-radius: 40px;
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  margin: auto;
+  max-width: 550px;
+  height: auto;
+  text-align: center;
+  flex-wrap: wrap;
+}
+
+.footer-logo-item {
+  color: #F5F7FA;
+  width: 160px;
+  height: 70px;
+  border-radius: 20px;
+  background-color: #367694;
+  padding: 10px;
+  margin: 7.5px;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.footer-logo-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  
+}
+
+.footer-logo-image {
+  width: 150px;
+  /* height: 50px; */
```

Comment:
Avoid commented-out code in the pull request.

Reply:
updates:
1. removed commented-out code
2. updated css to force image size to fit in box
3. removed <footer> in html pages and replaced with footer.js component in every html page
4. updated footer content to read 2023 copyright, not 2022 copyright